pkg/netceptor/netceptor.go

```diff
@@ -235,12 +245,13 @@ type WorkCommand struct {
 
 // ServiceAdvertisement is the data associated with a service advertisement.
 type ServiceAdvertisement struct {
-	NodeID       string
-	Service      string
-	Time         time.Time
-	ConnType     byte
-	Tags         map[string]string
-	WorkCommands []WorkCommand
+	NodeID        string
+	Service       string
+	Time          time.Time
+	ConnType      byte
+	ConnTypeLabel string
```

Comment:
This probably shouldn't go here.  This is the struct for the low-level advertisements that get sent around the network, so its size matters.  If we want another field in the Status response, we should define another struct that inherits this one and adds the field, and populate it at the time the status is being asked for.